ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Extensions/ObjectExtensions.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace ExadelTimeTrackingSystem.BusinessLogic.Extensions
+{
+    using System.Text.Json;
+
+    public static class ObjectExtensions
```

Comment:
better to move it to Data layer, it's common for all types of objects and can be useful for models